### PR TITLE
feat(graphql): exported all graphql types

### PR DESCRIPTION
**Pull request by osherv**
## Which problem is this PR solving?

Users who want to configure GraphQLInstrumentationExecutionResponseHook, can't use this type.

## Short description of the changes

export all types from graphql instrumentation.

## Checklist

- [x] Ran `npm run test-all-versions` for the edited package(s) on the latest commit if applicable.

### CODE REVIEW

File: plugins/node/opentelemetry-instrumentation-graphql/src/index.ts [hunk 1]
```diff
@@ -15,4 +15,4 @@
  */
 
 export * from './instrumentation';
-export { GraphQLInstrumentationConfig } from './types';
+export * from './types';
```
Comment (blumamir) [inline plugins/node/opentelemetry-instrumentation-graphql/src/index.ts:18]: generally LGTM, but the `types.ts` file exports many many internal stuff which are not useful to the instrumentation users.
WDYT about additionally exporting just the `GraphQLInstrumentationExecutionResponseHook` and not everything?

Comment (osherv) [reply plugins/node/opentelemetry-instrumentation-graphql/src/index.ts]: Fixed, Thanks!

**Review by Rauno56 — APPROVED** [review]

**Comment by Rauno56** [comment]
Would highly recommend you to create future PRs off of a personal fork not one belonging to a org, @osherv.

**Comment by osherv** [comment]
@Rauno56 Thanks! I'll start using my own fork!

**Review by vmarchaud — APPROVED** [review]

**Pull request merged**